Microsoft.Azure.Cosmos.Encryption/src/EncryptionContainer.cs

```diff
@@ -140,20 +129,14 @@ internal sealed class EncryptionContainer : Container
             ItemRequestOptions requestOptions = null,
             CancellationToken cancellationToken = default)
         {
-            CosmosDiagnosticsContext diagnosticsContext = CosmosDiagnosticsContext.Create(requestOptions);
-            using (diagnosticsContext.CreateScope("ReadItem"))
-            {
-                ResponseMessage responseMessage;
-
-                responseMessage = await this.ReadItemHelperAsync(
-                    id,
-                    partitionKey,
-                    requestOptions,
-                    diagnosticsContext,
-                    cancellationToken);
+            ResponseMessage responseMessage;
+            responseMessage = await this.ReadItemHelperAsync(
```

Comment:
nit: no need to put it on a separate line
```suggestion
            ResponseMessage responseMessage = await this.ReadItemHelperAsync(
```

Reply:
Fixed